into/backends/sparksql.py

```diff
@@ -57,7 +65,16 @@ def rdd_to_sqlcontext(ctx, rdd, name=None, dshape=None, **kwargs):
     if isdimension(dshape.parameters[0]):
         dshape = dshape.measure
     sql_schema = dshape_to_schema(dshape)
-    sdf = ctx.applySchema(rdd, sql_schema)
+
+    try:
+        sdf = ctx.applySchema(rdd, sql_schema)
+    except TypeError:
+        # TODO: gross
+        # we have to convert a tuple of strings to have typed elements
+        # this seems like it would be horribly inefficient
+        rdd = rdd.map(lambda x: convert(np.ndarray, [x], dshape=dshape).item())
```

Comment:
Is SparkSQL able to make these coercions?

In the future when there is more time we could also consider make a Pure Python function that transformed a datashape into a function to do a conversion like this.  I'd want to wait until someone asked for this before spending time on it though.